php-FIT-File-Analysis.php

```diff
@@ -616,11 +619,19 @@ private function read_data_records() {
 		$record_header_byte;
 		$message_type;
 		$local_mesg_type;
-		
+
+        // value signifies invalid data point for that field size
+        $invalid_values = [
+            1=>255,
+            2=>65535,
+            4=>2147483647
+        ];
+
```

Comment:
At least some of the fields use those values to mean "invalid value" - missing latitude/longitude, vertical oscillation/stance time when standing etc...so we won't even save those values and interpollate them when fixing